### PR TITLE
add utf-8 food

### PR DESCRIPTION
**Pull request by clouserw**
I thought we could use some utf-8 food

### CODE REVIEW

**Comment by chuckharmston** [comment]
r- doesn't weight burrito sufficiently high.

**Comment by chuckharmston** [comment]
(but r+ really)

**Comment by washort** [comment]
r+ them's good eatin'

**Comment by clouserw** [comment]
@washort it looks like the tests are finding a legitimate failure but I don't know why they are failling on my PR and not already...

File: mkt/webapps/fakedata.py [hunk 1]
```diff
@@ -29,11 +30,11 @@
 from mkt.webapps.models import AddonUser, AppManifest, Preview
 
 adjectives = ['Exquisite', 'Delicious', 'Elegant', 'Swanky', 'Spicy',
-              'Food Truck', 'Artisanal', 'Tasty']
+              'Food Truck', 'Artisanal', 'Tasty', 'Questionable', u'Drôle']
 nouns = ['Sandwich', 'Pizza', 'Curry', 'Pierogi', 'Sushi', 'Salad', 'Stew',
          'Pasta', 'Barbeque', 'Bacon', 'Pancake', 'Waffle', 'Chocolate',
-         'Gyro', 'Cookie', 'Burrito', 'Pie']
-fake_app_names = list(itertools.product(adjectives, nouns))[:-1]
```
Comment (washort) [inline mkt/webapps/fakedata.py]: this was just my little joke to keep "Tasty Pie" from being generated. :-)

File: mkt/webapps/tests/test_fakedata.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ def test_smallset(self):
         ok_(not any(appname[-1].isdigit() for appname, cat in data))
 
     def test_bigset(self):
-        size = 300
+        size = 1500
```
Comment (washort) [inline mkt/webapps/tests/test_fakedata.py]: Good idea, this should work just fine now that I've simplified things a bit; the original version of this code was mainly intended for small sets so I didn't look too close at the bigger cases.

**Comment by washort** [comment]
r+

**Comment by clouserw** [comment]
Thanks!

**Pull request merged**